src/session_impl.cpp

```diff
@@ -6599,10 +6614,19 @@ namespace {
 		if (m_natpmp) m_natpmp->delete_mapping(handle);
 	}
 
+	void session_impl::stop_ip_notifier()
+	{
+		if (!m_ip_notifier) return;
+
+		m_ip_notifier->cancel();
+		m_ip_notifier.reset();
+	}
+
 	void session_impl::stop_lsd()
 	{
-		if (m_lsd)
-			m_lsd->close();
+		if (!m_lsd) return;
```

Comment:
this change should be no necessary, but something happened in `stop_ip_notifier`, waiting for CI jobs to finish/fail again